seatunnel-server/seatunnel-app/src/main/resources/application.yml

```diff
@@ -38,6 +44,12 @@ jwt:
   secretKey:
   algorithm: HS256
 
+seatunnel:
```

Comment:
Since this configuration is specifically for the seatunnel-web, placing it under seatunnel can be confusing. Could you please remove the word "seatunnel" and move the configuration under "spring"?